src/app/screens/restoreFunds/restoreBtc/index.tsx

```diff
@@ -130,7 +130,7 @@ function RestoreBtc() {
   const onClickTransfer = () => {
     mutateSignNonOrdinalBtcTransaction({
       recipientAddress: btcAddress,
-      nonOrdinalUtxos: unspentUtxos,
+      nonOrdinalUtxos: unspentUtxos ?? [],
```

Comment:
I don't think we really want to fire this event if unspentUtxos is undefined. Maybe something like this rather?

```suggestion
const onClickTransfer = () => {
  if (!unspentUtxos) return;
    mutateSignNonOrdinalBtcTransaction({
      recipientAddress: btcAddress,
      nonOrdinalUtxos: unspentUtxos,
```

Reply:
We might also want to disable the button and show a loader while unspentUtxos is undefine